src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -48,6 +67,31 @@ export const SueReportLocation = ({
   const zipCodeInputRef = useRef();
   const cityInputRef = useRef();
 
+  const queryVariables = {
+    start_date: '1900-01-01T00:00:00+01:00',
+    status: `${SueStatus.UNPROCESSED},${SueStatus.OPEN},${SueStatus.IN_PROCESS},${SueStatus.WAIT_REQUESTOR},${SueStatus.WAIT_THIRDPARTY}`
```

Comment:
wouldn't it make more sense to have an object and iterate over it to place all the values here? it is not very practicable to create a string and manually put every key of the object here. how is this beneficial?

Reply:
Yes, I think it makes more sense. It was not right to write with text. I updated the code with map function. 

https://github.com/smart-village-solutions/smart-village-app-app/pull/1059/commits/5d5f56afca33f876b5e8509dd22ab4bf64a81014